BackofficeBundle/Resources/views/BackOffice/Include/mainPanel.html.twig

```diff
@@ -2,10 +2,13 @@
 <div id="main" role="main">
 
     {% include 'OpenOrchestraBackofficeBundle:BackOffice/Include/MainPanel:ribbon.html.twig' %}
-
+	{% set alert %}
+	    {% include 'BraincraftedBootstrapBundle::flash.html.twig' %}
+	{% endset %}
```

Comment:
It's a good idea to add the flashbag rendering in this part of the dom, but can you do it via a js mechanism ? I mean that way it works only (if I'm not mistaken) when the entire page is refreshed. But if it was a js mechanism, we could use it anytime we need it without refreshing the page.